include/ebpf_nethooks.h

```diff
@@ -265,6 +265,24 @@ typedef struct _bpf_sock_ops
 typedef int
 sock_ops_hook_t(bpf_sock_ops_t* context);
 
+typedef enum _process_operation
+{
+    PROCESS_OPERATION_CREATE, ///< Process creation.
+    PROCESS_OPERATION_DELETE, ///< Process deletion.
+} process_operation_t;
+
+typedef struct _process_md
+{
+    uint8_t* command_start;        ///< Pointer to start of the command line.
+    uint8_t* command_end;          ///< Pointer to end of the command line.
+    uint64_t process_id;           ///< Process ID.
+    uint64_t parent_process_id;    ///< Parent process ID.
+    uint64_t creating_process_id;  ///< Creating process ID.
+    uint64_t creating_thread_id;   ///< Creating thread ID.
+    uint64_t creation_status;      ///< Process creation status.
```

Comment:
What does this mean to be a uint64_t?  I.e. what would the value "5" mean?   Shouldn't this be an enum or something more understandable?

Reply:
Thanks. I will improve the documentation. Here is what the MSDN says:
https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/ntddk/ns-ntddk-_ps_create_notify_info

> CreationStatus
> 
> The NTSTATUS value to return for the process-creation operation. Drivers can change this value to an error code to prevent the process from being created.

Reply:
Wanted to switch this to NTSTATUS but there isn't a good way to do this without conflict with user mode and kernel mode headers.